bouncer/src/repo/core/model/bson/repo_bson_ref.cpp

```diff
@@ -34,10 +34,14 @@ std::string RepoRef::convertTypeAsString(const RefType &type) {
 	}
 }
 
-std::string RepoRef::getFileName() const {
+std::string RepoRef::getID() const {
```

Comment:
Ironically the only place this is used is in the unit test... where it is [set to a string that is not a UUID](https://github.com/3drepo/3drepobouncer/blob/7f001fd8c1b125f318ee73c9df46348d2d44d60e/test/src/unit/repo/core/model/bson/ut_repo_bson_ref.cpp#L37)! 🙈

I have updated it to check for the type and added a unit test anyway though, in case someone does need it in the future and makes the same assumption..